xbmc/utils/log.h

```diff
@@ -42,7 +42,7 @@ class CLog
   CLog();
   ~CLog(void);
   static void Close();
-  static void Log(int loglevel, PRINTF_FORMAT_STRING const char *format, ...) PARAM2_PRINTF_FORMAT;
+  static void Log(int loglevel, PRINTF_FORMAT_STRING const char *format, ...);
```

Comment:
This should probably use a variadic template now that it passes everything on to libfmt but that's a larger change.